docs/quick-start.md

```diff
@@ -0,0 +1,216 @@
+---
+name: Quick Start
+---
+
+# Quick Start
```

Comment:
```suggestion
# Quick Start
```

None of the other docs use frontmatter and the title is the same value as the name.